### DISCUSSION TITLE
Parameter Identification example

### DISCUSSION THREAD

**Discussion by jgonlop3** [Open-Systems-Pharmacology/Forum#1877] [Questions & Problems]
In the OPS manual, an example of parameter identification is given with a file called Theophylline.pksim5.

Where can this file be downloaded?

**Comment by rwmcintosh** [comment]
The examples referred to were removed from the PK-Sim installer.

The repository can be found [Example_Theophylline](https://github.com/Open-Systems-Pharmacology/Example_Theophylline)

Documentation [issue](https://github.com/Open-Systems-Pharmacology/docs/issues/242)
